patches/chrome-browser-resources-downloads-toolbar.js.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/chrome/browser/resources/downloads/toolbar.js b/chrome/browser/resources/downloads/toolbar.js
+index 4e674024d6f305881a0b64ebacf582c1433671f1..06f73ebff7e5de4a08d33b18892b35e1e2901e99 100644
+--- a/chrome/browser/resources/downloads/toolbar.js
++++ b/chrome/browser/resources/downloads/toolbar.js
+@@ -4,7 +4,7 @@
+ 
+ import 'chrome://resources/cr_elements/cr_action_menu/cr_action_menu.m.js';
+ import 'chrome://resources/cr_elements/cr_icon_button/cr_icon_button.m.js';
+-import 'chrome://resources/cr_elements/cr_toolbar/cr_toolbar.m.js';
```

Comment:
same as toolbar, we should follow-up to replace the file using a chromium_src override on the generated resource file

Reply:
and actually there are several of these patches which makes no sense because at the very least we should just be patching `cr_toolbar.m.js` to import `chrome://brave-resources/br_elements/br_toolbar/br_toolbar.m.js`